### PR TITLE
Add no-eol keyword; write .err files only for failed suites.

### PR DESCRIPTION
**Pull request by dtcaciuc**

### CODE REVIEW

File: grill/runner.go [hunk 1]
```diff
@@ -115,15 +115,20 @@ func (t *Test) Run(ctx TestContext) error {
...
-		if ok {
+		if exitErr, ok := err.(*exec.ExitError); ok {
 			buf.Write(exitErr.Stderr)
 			status := exitErr.Sys()
 			if s, ok := status.(syscall.WaitStatus); ok {
 				fmt.Fprintf(buf, "[%d]", s.ExitStatus())
 			}
 			err = nil
 		}
+		// TODO else?
```
Comment (echlebek) [inline grill/runner.go]: Docs say other errors can be returned, but this is pretty fatal, IMO panic.

File: test.sh [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+
 go test .
 
+export PATH=`pwd`:$PATH
 export LANG=C
+
 rm -f examples/*.err
-./cram --quiet examples/env.t
+
+cram examples/*.t tests/*.t
```
Comment (echlebek) [inline test.sh]: Why cram? Shouldn't this be grill?

Comment (dtcaciuc) [reply test.sh]: Yeah, I it should. I originally named it cram to keep the test suite as unchanged as possible. There's also a clash between output grill binary and grill/ directory. Should I move exec sources to standard cmd/grill/ and move grill/ contents back to root?

File: grill/runner.go (hunk 1 above)
Comment (dtcaciuc) [reply grill/runner.go]: Done

File: test.sh (hunk 2 above)
Comment (echlebek) [reply test.sh]: Yeah, let's put the command in cmd/grill

Comment (dtcaciuc) [reply test.sh]: Done

File: cmd/grill/main.go [hunk 3]
```diff
@@ -6,7 +6,7 @@ import (
 	"log"
 	"os"
 
-	"github.com/echlebek/grill/grill"
+	grill "github.com/echlebek/grill/internal"
```
Comment (echlebek) [inline cmd/grill/main.go]: Better to have a subdirectory in internal called grill, internal protects the entire subtree from being imported.

Comment (dtcaciuc) [reply cmd/grill/main.go]: PTAL

**Review by echlebek — APPROVED** [review]
:shipit:

**Pull request merged**